index.html

```diff
@@ -36,134 +38,127 @@
+            The <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd</a> office has an event space that you
+            can use for community events. It has a capacity of 90 people (seated) or 30 people (with desks for courses).
           </p>
         </div>
-        <div>
-          <h2>Supporters</h2>
+        <div class="speaker">
+          <h2 class="header-style">Meet our React experts</h2>
           <p>
-            Our Meetups are hosted by <a href="http://bitcrowd.net">bitcrowd.net</a>.
-            The template for this page is the gloriuos <a href="http://berlinjs.org/" title="Berlin's finest JavaScript Usergroup">berlinjs.org page</a> with kind permission of <a href="http://www.twitter.com/rmehner" title="Follow Robin on Twitter">Robin Mehner</a>.
+            We at <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd</a> have created many ReactJS applications 
+            for our clients over the past few years. Just drop us an email or give us a call and we’ll see how we can help 
+            your project.
           </p>
         </div>
       </div>
-      <section>
+      <section class="clearfix three-cols">
+          <div itemscope="" itemtype="http://schema.org/Organization">
+            <h3>Get in touch</h3>
+            <ul>
+              <li itemprop="email">
+                <a href="mailto:info@bitcrowd.net?Subject=Hallo bitcrowd!">
+                  <span>
+                   Email: info@bitcrowd.net
+                  </span>
+                </a>
+              </li>
+              <li itemprop="twitter">
+                <a href="https://twitter.com/bitcrowd">
+                  <span>
+                    Twitter: @bitcrowd
+                  </span>
+                </a>
+              </li>
+              <li itemprop="telephone">
+                <a href="tel:493055577597">
+                  <span>
+                    +49 30 555 77 597
+                  </span>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <h3>Come visit us at:</h3>
+            <span itemprop="name">bitcrowd GmbH.</span>
+            <address itemprop="address" itemtype="http://schema.org/PostalAddress">
+              <span itemprop="streetAddress">Oranienstr. 6</span>
+              <br>
+              <span itemprop="postalCode">10997</span>
+              <span itemprop="addressLocality">Berlin<br>Germany</span>
+            </address>
+            <a href="https://www.google.de/maps/place/Oranienstra%C3%9Fe+6,+10997+Berlin/@52.4998919,13.4244531,18z/data=!3m1!4b1!4m5!3m4!1s0x47a84e350a669333:0x3dc79e96d74bd57b!8m2!3d52.4998919!4d13.425056">
+              Open in google maps</a>
+          </div>
+      </section>
+    </main>
+
+    <footer class="content">
+    <section>
```

Comment:
Which `section` are you referring to, the one in the `footer` or this one?

     <section class="clearfix three-cols">
          <div itemscope="" itemtype="http://schema.org/Organization">
            <h3>Get in touch</h3>
            <ul>
              <li itemprop="email">
                <a href="mailto:info@bitcrowd.net?Subject=Hallo bitcrowd!">
                   info@bitcrowd.net
                </a>
              </li>
              <li itemprop="twitter">
                <a href="https://twitter.com/bitcrowd">
                    @bitcrowd
                </a>
              </li>
              <li itemprop="telephone">
                <a href="tel:493055577597">
                    +49 30 555 77 597
                </a>
              </li>
            </ul>
          </div>
          <div>
            <h3>Come visit us at:</h3>
            <span itemprop="name">bitcrowd GmbH.</span>
            <address itemprop="address" itemtype="http://schema.org/PostalAddress">
              <span itemprop="streetAddress">Oranienstr. 6</span>
              <br>
              <span itemprop="postalCode">10997</span>
              <span itemprop="addressLocality">Berlin<br>Germany</span>
            </address>
            <a href="https://www.google.de/maps/place/Oranienstra%C3%9Fe+6,+10997+Berlin/@52.4998919,13.4244531,18z/data=!3m1!4b1!4m5!3m4!1s0x47a84e350a669333:0x3dc79e96d74bd57b!8m2!3d52.4998919!4d13.425056">
              Open in google maps</a>
          </div>
          <div>
            <h3>Supporters</h3>
            <p>
              Our Meetups are hosted by <a href="http://bitcrowd.net">bitcrowd.net</a>.
              The template for this page is the gloriuos <a href="http://berlinjs.org/" title="Berlin's finest JavaScript Usergroup">berlinjs.org page</a> with kind permission of <a href="http://www.twitter.com/rmehner" title="Follow Robin on Twitter">Robin Mehner</a>.
            </p>
          </div>
          <div>
            <h3>The team</h3>
            <p>
              ReactJS.berlin was organized by
              <a href="http://www.twitter.com/chris_bitcrowd" title="Follow Chris on Twitter">Christoph Beck</a> from <a href="http://www.twitter.com/bitcrowd" title="Follow bitcrowd on Twitter">@bitcrowd</a>. Interested in joining the organizers team? Drop us a line on twitter or meetup :) Design by <a href="http://www.twitter.com/m_besser" title="Follow Matti on Twitter">Matti Besser</a> &amp; <a href="http://www.twitter.com/plankton" title="Follow Darren on Twitter">Darren Cadwallader</a>.
            </p>
          </div>
      </section>

 Remove the `div` that contains the `Get in touch` content?

Reply:
I’m referring to the one in the line this comment is attached to :D that’s in the footer I think. Right now it looks like this:

```html
<footer class="content">
    <section>
        <div class="clearfix one-col">
          <div>
            <h2>Past Meetups</h2>
          </div>
        </div>
      </section>
</footer>
```

That’s a lot of unnecessary `<div>`s, it could be:

```html
<footer class="content">
    <section class="clearfix one-col">
    </section>
</footer>
```

Reply:
Okay, so I managed to remove one `div` (there were 2 that could potentially be deleted) and I moved the `class="clearfix one-col"` to the `<section>`.

There second (more inner) div is harder to remove because of this css rule:

```
.one-col > div,
.three-cols > div {
  width: 100%;
  padding: 10px;
}
```